Sim/SimDelphesInterface/src/ParticleStatus.h

```diff
@@ -0,0 +1,23 @@
+/** @class ParticleStatus
+ *
+ *  Lists possible statuses of simulated or reconstructed particles
+ *
+ *  @author: Z. Drasal (CERN)
+ *
+ */
+#ifndef INCLUDE_PARTICLE_STATUS_H_
+#define INCLUDE_PARTICLE_STATUS_H_
+
+namespace ParticleStatus {
```

Comment:
This is for testing purposes, one should use bit encoder/decoder instead. But yes, I agree enum is probably better.